### PR TITLE
add 28. Implement strStr

### PR DESCRIPTION
**Pull request by katsurada1**
Please check my code.

### CODE REVIEW

**Comment by kobakobashu** [comment]
empty file

**Comment by katsurada1** [comment]
I am sorry, I had accidentally deleted the file.
Please check my code.

**Comment by katsurada1** [comment]
> I am sorry, I had accidentally deleted the file.
> Please check my code.

**Comment by katsurada1** [comment]
> empty file

I am sorry, I had accidentally deleted the file.
Please check my code.

File: 28.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    def strStr(self, haystack: str, needle: str) -> int:
+        """A function to return the index of the first occurrence of needle in haystack.
+        Args:
+            haystack (str): A given a string.
+            needle (str): A given a string.
+        Returns:
+            int: The index of the first occurrence of needle in haystack, or -1 if needle is not part of haystack, or 0 if needle is an empty string.
+        """
+        
+        if len(needle) == 0:
```
Comment (kobakobashu) [inline 28.py:16]: if not needle
https://qiita.com/simonritchie/items/bb06a7521ae6560738a7#%E6%9D%A1%E4%BB%B6%E6%96%87%E3%81%AE%E6%9B%B8%E3%81%8D%E6%96%B9

File: 28.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# n = len(haystack) - len(needle) + 1
```
Comment (kobakobashu) [inline 28.py:1]: n = len(haystack)

**Review by kobakobashu — CHANGES_REQUESTED** [review]
👀

**Comment by katsurada1** [comment]
I modified the code.
Please check my code.

**Review by kobakobashu — APPROVED** [review]
LGTM

**Pull request merged**